native/src/routes/SearchModal.tsx

```diff
@@ -53,8 +54,9 @@ const SearchModal = ({
   cityCode,
   closeModal,
   t,
+  initialSearchText = '',
 }: SearchModalProps): ReactElement => {
-  const [query, setQuery] = useState<string>('')
+  const [query, setQuery] = useState<string>(initialSearchText ?? '')
```

Comment:
Setting the default twice is unnecessary. Also if you apply my changes above, by setting the `initalSearchText` in the SearchModalContainer, you do not need to do it here and can adjust the type of the prop to `initialSearchText: string` as neigher `undefined` nor `null` is possible